html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm

```diff
@@ -199,6 +199,37 @@ has_field 'autoregister' =>
              help => 'This activates automatic registation of devices for the profile. Devices will not be shown a captive portal and RADIUS authentication credentials will be used to register the device. This option only makes sense in the context of an 802.1x authentication.' },
   );
 
+=head2 ipsk
+
+Controls whether or not this connection profile to enabled ipsk
+
+=cut
+
+has_field 'ipsk' =>
+  (
+   type => 'Toggle',
+   label => 'Enable or not ipsk',
+   checkbox_value => 'enabled',
+   unchecked_value => 'disabled',
+   default => 'disabled',
+   tags => { after_element => \&help,
+             help => 'This activates ipsk feature on this connection profile. It mean that radius will answer specific attributes like the psk key to use to connect on the SSID.' },
+  );
+
+=head2 default_psk_key
+
+Define the default psk key to connect on this connection profile
+
+=cut
+
+has_field 'default_psk_key' =>
+  (
+   type => 'Text',
+   label => 'Default psk key',
+   tags => { after_element => \&help,
+             help => 'This is the default psk key when you enable ipsk on this connection profile.length of 8 minimum' },
```

Comment:
This is the default PSK key when you enable IPSK on this connection profile. The minimum length is eight characters.